ntoskrnl/ex/work.c

```diff
@@ -262,13 +263,19 @@ ExpCreateWorkerThread(WORK_QUEUE_TYPE WorkQueueType,
     if (Dynamic) Context |= EX_DYNAMIC_WORK_THREAD;
 
     /* Create the System Thread */
-    PsCreateSystemThread(&hThread,
-                         THREAD_ALL_ACCESS,
-                         NULL,
-                         NULL,
-                         NULL,
-                         ExpWorkerThreadEntryPoint,
-                         UlongToPtr(Context));
+    Status = PsCreateSystemThread(&hThread,
+                                  THREAD_ALL_ACCESS,
+                                  NULL,
+                                  NULL,
+                                  NULL,
+                                  ExpWorkerThreadEntryPoint,
+                                  UlongToPtr(Context));
+    if (!NT_SUCCESS(Status))
+    {
+        /* Well... */
+        DPRINT1("Failed to crate worker thread: 0x%08x\n", Status);
```

Comment:
```suggestion
        DPRINT1("Failed to create worker thread: 0x%08x\n", Status);
```

Reply:
It's strange that there's no complaint on the contrary for the `ObReferenceObjectByHandle` call above where no return value check has been done whatsoever.

Reply:
Because nobody added the `_Must_inspect_result_` annotation to the declaration of `ObReferenceObjectByHandle`